lib/parsers/asm-parser.js

```diff
@@ -37,7 +37,7 @@ export class AsmParser extends AsmRegex {
         this.labelFindMips = /[$.A-Z_a-z][\w$.]*/g;
         this.mipsLabelDefinition = /^\$[\w$.]+:/;
         this.dataDefn = /^\s*\.(string|asciz|ascii|[1248]?byte|short|x?word|long|quad|value|zero)/;
-        this.fileFind = /^\s*\.file\s+(\d+)\s+"([^"]+)"(\s+"([^"]+)")?.*/;
+        this.fileFind = /^\s*\.(?:cv_)?file\s+(\d+)\s+"([^"]+)"(\s+"([^"]+)")?.*/;
```

Comment:
while you're here can you fix #4036 on line 44 (GH won't let me comment on a unchaanged line). probably needs to be `[\dA-Z_a-z]`)

Reply:
Done! Left it as `[$.@A-Z_a-z]\w*`